config/default.yml

```diff
@@ -209,6 +209,11 @@ SketchupRequirements/GlobalVariables:
   Reference: https://github.com/SketchUp/rubocop-sketchup/tree/main/manual/cops_requirements.md#globalvariables
   Enabled: true
 
+SketchupRequirements/InitEntity:
+  Description: Do not init SketchUp Entity objects with `new`.
```

Comment:
I'd expand `init` to `initialize`.

Reply:
Does this need to be regenerated? `bundle exec rake` ?

Reply:
Ah, missed the one in the description